docs/source/cpp/sbml.rst

```diff
@@ -0,0 +1,65 @@
+Creating models using SBML
+===========================
+
+The `SBML <https://sbml.org/>`_ integration into memilio works via the ``sbml2memilio`` executable. 
+Building it requires the installation of `libsbml <https://sbml.org/software/libsbml/>`_ . Then the general build 
+command will produce the executable in the `build/bin`-directory. 
+
+Building the converter
+---------------------------------
+
+To build the executable, the following steps are necessary:
+
+1. Install ``libsbml``
```

Comment:
write a bit more here. Maybe just "using the provided link above. Or move this part into this step here. At least provide the link